packages/auth-ts/package.json

```diff
@@ -66,6 +67,7 @@
   },
   "dependencies": {
     "@firebase/util": "^0.2.36",
+    "@react-native-community/async-storage": "1.9.0",
```

Comment:
the original SDK doesn't have this as a dependency, can you see how they did it?

Reply:
They have the deprecated version of this as a dependency

Reply:
is it specified somewhere other than https://github.com/firebase/firebase-js-sdk/blob/master/packages/auth/package.json?

Reply:
Is what specified?

Reply:
Oh so this is my bad: This shouldn't be in package.json regardless. I didn't realize you were commenting on package.json